modules/book.js

```diff
@@ -0,0 +1,48 @@
+export default class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+      bookList = [];
+
+      add() {
+        if (document.getElementById('title').value && document.getElementById('author').value) {
+          const temp = {
+            title: document.getElementById('title').value,
+            author: document.getElementById('author').value,
+          };
+          this.bookList.push(temp);
```

Comment:
[OPTIONAL] I suggest you make use of concat() instead of push() since push mutates the array😉.